### PR TITLE
popup done

### PR DESCRIPTION
**Pull request by Oodmincheg**
### Google popup

[Demo](https://oodmincheg.github.io/ggl-popup/) | [Code base](https://github.com/Oodmincheg/ggl-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/Oodmincheg/google-popup/style.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+.icon {
+  max-width: 84px;
+  max-height: 84px;
+  padding: 14px; }
+  .icon:hover {
+    border: 0.5px lightgray solid; }
+  .icon:focus {
+    border: 0.5px blue solid; }
+
+#open-modal-second-label {
```
Comment (A-Ostrovnyy) [inline submissions/Oodmincheg/google-popup/style.css]: Avoid styling by `#id`.

Comment (Oodmincheg) [reply submissions/Oodmincheg/google-popup/style.css]: Added classes selectors.

**Comment by Oodmincheg** [comment]
> Good work!
> Every interactive element should have `:hover` and `:focus` state, `:active` optional(good practice add it too).
> Missing `:focus` on the menu icon, bell icon, and user avatar. Your menu must be accessible from the keyboard. When the menu icon on focus I press Space or Enter key and popup must open. Same story with More button inside the popup.

Done. @A-Ostrovnyy, Please review

**Comment by A-Ostrovnyy** [comment]
Good job!
Bell icon and user avatar still without outline/border on `:focus` state. Add `tabindex` attribute to this `li` element

**Pull request merged**